torch/csrc/jit/python/pybind_utils.h

```diff
@@ -1038,6 +1039,26 @@ inline py::object invokeScriptMethodFromPython(
       });
 }
 
+inline c10::optional<Method> matchOverloadedMethods(
+    Method& method,
+    const struct tuple_slice& args,
+    const pybind11::kwargs& kwargs) {
+  auto methods = method.owner().get_overloaded_methods(method.name());
+  for (auto method : methods) {
+    try {
+      createStackForSchema(
+          method.function().getSchema(),
+          args,
+          kwargs,
+          method.owner()._ivalue());
+      return method;
+    } catch (...) {
```

Comment:
I think we should only catch `schema_match_error` here rather than any generic exception

Reply:
done